docs/uploading_funding_csvs.md

```diff
@@ -0,0 +1,85 @@
+# Importing funding data from CSVs
+
+## What is a trainee summary?
+
+Trainee summaries contain (for each lead school or provider) the number of
+trainees receiving different types of funding, split by training route and
+subject.
+
+## What is a payment schedule?
+
+Payment schedules contain (for each lead school or provider) the total amount
+of funding they are expected to receive each month, split by funding type.
+
+Some of the months will contain actual amounts if funding has already been paid.
+The rest will contain predicted amounts.
+
+## Files we receive
+
+Every month the funding team send us four funding-related CSVs. They will be
+named something like this:
+
+* `SDS_subject_breakdown_{Month}.csv`
+  * The trainee summary for lead schools.
+* `SDS_Profile_{Month}.csv`
+  * The payment schedule for lead schools.
+* `TB_summary_upload_{Month}.csv`
+  * The trainee summary for providers.
+* `TB_Profile_{Month}.csv`
+  * The payment schedule for providers.
+
+For the payment schedules, we will also be told which months contain predicted
+values. If this is not clear, then reach out to the funding team to clarify.
+
+## How to import the data
+
+### Importing trainee summaries
+
+1. Download the trainee summary file e.g. 'SDS_subject_breakdown_January.csv'
+
+2. Copy the CSV to your clipboard
+
+    ```bash
+    cat ~/Desktop/SDS_subject_breakdown_January.csv | pbcopy
+    ```
+
+3. Check that you've definitely copied the correct thing! Paste it somewhere
+
+4. SSH onto register-worker-production, navigate into the app directory and
+  create a new file
+
+    ```bash
+    cf ssh register-worker-production
+    cd /app
```

Comment:
Even though we delete the file a bit later `/app/tmp` would probably be a better place for it.

Reply:
Updated!